src/analytics/README.md.txt

```diff
@@ -0,0 +1,26 @@
+fetch branch from github.
+
+create local python environment
+  windows
+	python -m venv [local_env_name e.g env]
+	
+   linux/mac
+	python3 -m venv [local_env_name e.g env]
+
+activate the environment
+	windows
+	- [local_env_name]\Scripts\activate
+	
+	linux/mac
+	- source [local_env_name]/bin/activate
+
+Install dependencies using the requirements.txt
+	-pip install -r requirements.txt
+
+set environment variables
+ windows
+ - set FLASK_APP=app.py
+ - set FLASK_ENV=development
+
+Run the Flask App
+    - flask run
```

Comment:
-Rename the file to remove the txt extension
- add the option for setting up the env variables for linux/mac

Reply:
looks good

Reply:
resolved